collect_app/src/main/java/org/odk/collect/android/activities/viewmodels/MainMenuViewModel.java

```diff
@@ -0,0 +1,28 @@
+package org.odk.collect.android.activities.viewmodels;
+
+import androidx.annotation.Nullable;
+import androidx.lifecycle.ViewModel;
+
+import org.odk.collect.android.BuildConfig;
+
+public class MainMenuViewModel extends ViewModel {
+
+    public String getVersion() {
+        return versionNameWithoutPrefix().split("-")[0];
+    }
+
+    @Nullable
+    public String getVersionSHA() {
+        String[] split = versionNameWithoutPrefix().split("-", 2);
+
+        if (split.length > 1) {
+            return split[1];
+        } else {
+            return null;
+        }
+    }
+
+    private String versionNameWithoutPrefix() {
+        return BuildConfig.VERSION_NAME.split("v")[1];
```

Comment:
How did you decide to remove the `v`? We have the `v` prefix everywhere that we show a Collect version including GitHub releases, the forum, the Play Store release notes, etc. That's the case across all ODK tools. I like the consistency and would personally keep the `v` in.

Reply:
Yeah no strong feeling really. Will change this.